lefthook.yml

```diff
@@ -0,0 +1,11 @@
+pre-commit:
+  parallel: true
+  commands:
+    lint:
```

Comment:
Perhaps test could be added, but I would like to introduce only lint first and see how it goes.